src/instructlab/sdg/generate_data.py

```diff
@@ -234,6 +238,7 @@ def load_pipeline(yaml_basename):
         SDG([load_pipeline("knowledge.yaml")]),
         SDG([load_pipeline("freeform_skills.yaml")]),
         SDG([load_pipeline("grounded_skills.yaml")]),
+        SDG([load_pipeline("mmlu_bench.yaml")]),
```

Comment:
I don't really like this ... to date, the 3 YAML files that we require for a "pipeline set" correspond to the 3 types of data generation we support with the current taxonomy format. See e.g. #134

This MMLU bench pipeline is orthogonal - AIUI, you can use it on a knowledge training set from any knowledge pipeline to generate data that can be used for evaluation?

I'd be tempted to put it into `pipelines/eval` ?

Reply:
And something like:

```
def init_pipelines():
    training_data_pipelines = {
        'knowledge': ...,
        'freeform_skills': ...,
        'grounded_skills': ...,
    }
    eval_data_pipelines = {
       'mmlu_bench': ....,
    }
    return training_data_pipelines, eval_data_pipelines
```

Reply:
Nevermind this suggestion - I think we should treat loading `mmlu_bench.yaml` differently from the other cases

1. It's not in the same class of pipelines as the knowledge and skills pipelines - it is not part of a set of pipelines that generates training data. It's a new type of pipeline - an evaluation data pipeline
2. I'm not aware of a use case where either downstream packaging or a user needs to add a custom mmlu bench pipeline - so we can load it from a fixed location, packaged in the wheel

Reply:
Makes sense, I didn't like it either and was expecting we'd be moving it as its special